src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import Home from './components/home';
+import CountryDetails from './components/countryDetailsPage';
+import { getCountries } from './redux/Home/HomeReducer';
+
+const App = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getCountries());
+  }, []);
```

Comment:
- [ ] on this section please notice that the `getCountries` function is causing the entire application not to run. I'm unable to check that you have addressed the other issues from the previous review because some changes you have made are currently not making the entire application run. please kindly address this and ask for another review.